lms/templates/commerce/provider.underscore

```diff
@@ -1,27 +1,26 @@
 <div class="provider-wrapper">
   <div class="provider-info">
-    <%= interpolate(
-          gettext("You still need to visit the %(display_name)s website to complete the credit process."),
-          { display_name: display_name }, true
+    <%- StringUtils.interpolate(
+          gettext("You still need to visit the {display_name} website to complete the credit process."),
+          { display_name: display_name }
         ) %>
   </div>
   <div class="provider-more-info">
-    <%= interpolate(
-          gettext("To finalize course credit, %(display_name)s requires %(platform_name)s learners to submit a credit request."),
-          { display_name: display_name, platform_name: platformName }, true
+    <%- StringUtils.interpolate(
+          gettext("To finalize course credit, {display_name} requires {platform_name} learners to submit a credit request."),
+          { display_name: display_name, platform_name: platformName }
         ) %>
   </div>
   <div class="provider-instructions">
-    <%= fulfillment_instructions %>
+    <%- edx.HtmlUtils.ensureHtml(fulfillment_instructions) %>
   </div>
 </div>
 
 <div class="provider-buttons-logos">
   <div class="provider-logo">
-    <%= interpolate("<img src='%s' alt='%s'></image>", [thumbnail_url, display_name]) %>
+    <img src="<%- thumbnail_url %>" alt="<%- display_name %>" />
```

Comment:
This is so much easier to read without ``interpolate``! It shows one of the benefits of doing this logic in Underscore and not in JavaScript.